nano/node/blockprocessor.cpp

```diff
@@ -455,7 +455,10 @@ nano::process_return nano::block_processor::process_one (nano::write_transaction
 		}
 		case nano::process_result::opened_burn_account:
 		{
-			node.logger.always_log (boost::str (boost::format ("*** Rejecting open block for burn account ***: %1%") % hash.to_string ()));
+			if (node.config.logging.ledger_logging ())
```

Comment:
always_log to try_log will decrease the log size in case of repeated attempts, I like it.